app/src/main/java/com/nextcloud/talk/controllers/ChatController.kt

```diff
@@ -1551,6 +1551,11 @@ class ChatController(args: Bundle) :
                 input.copyTo(output)
             }
         }
+        try {
+            fd.close()
+        } catch (e: IOException) {
+            Log.w(TAG, "Failed to close AssetFileDescriptor", e)
+        }
```

Comment:
Prefer `use` (see resources above) for cleaner code and safer execution; this will never be called if  the `copyTo` call throws an exception, for example.